conformance/base/admin_network_policy/core-egress-sctp-rules.yaml

```diff
@@ -70,3 +70,55 @@ spec:
         namespaceSelector:
           matchLabels:
             kubernetes.io/metadata.name: network-policy-conformance-hufflepuff
+  - name: "deny-to-professor-quirrell-everything"
```

Comment:
why did you have to add new rules here? Couldn't we reuse the existing rules and simply edit the yaml inside the test to add/delete/update labels?

Reply:
here and everywhere else -> I'd assume the base yamls don't have to change at all, it becomes huge if it keeps growing like this right? I'd think the test logic should be able to sort of wield in the existing rules and do what's needed.